### DISCUSSION TITLE
super comparatif

### DISCUSSION THREAD

**Discussion by evil512-dom** [JC144/EDF_Simulateur_Prix#32] [General]
J'ai longtemps cherché une app/site pour avoir ce comparatif.
Je l'ai utilisé et ça a confirmé mes choix.
Une amélioration possible serait de pouvoir saisir ses montants base ou hc/hp + abonnement de son fournisseur.

**Comment by JC144** [comment]
Bonjour,

En récupérant la solution en local, c'est tout à fait possible.

Je viens de rédiger une documentation pour cela : [https://github.com/JC144/EDF_Simulateur_Prix/wiki/Ajout-de-tarifs](https://github.com/JC144/EDF_Simulateur_Prix/wiki/Ajout-de-tarifs)
